src/arch/riscv/kernel/vspace.c

```diff
@@ -37,6 +39,7 @@
 #include <plat/machine/hardware.h>
 #include <kernel/stack.h>
 
+
```

Comment:
totally understandable about the whitespace only changes. We usually try to avoid those as well, This one must have slipped through the cracks. I'll make sure to remove it.